workshops/tic_tac_toe/README.md

```diff
@@ -0,0 +1,346 @@
+---
+name: 'Tic Tac Toe'
+description: 'Make the Classic Tic Tac Toe Game in Python'
+author: '@Prithul0218'
+---
+
+Tic Tac Toe is a game that most of us have played when we were bored in class. It's also a classic coding challenge when learning a new programming language. As complicated as it may seem at first, it's actually rather easy to code taking less than 80 lines.
```

Comment:
```suggestion
Tic Tac Toe is a game that most of us have played when we were bored in class. It's also a classic coding challenge when learning a new programming language. As complicated as it may seem at first, it's actually rather easy to code and takes less than 80 lines.
```